### DISCUSSION TITLE
TailwindCSS has no effect after installation.

### DISCUSSION THREAD

**Discussion by nodiscardme** [tailwindlabs/tailwindcss#8093] [Help]
I have installed tailwindcss via npm (using postcss) and configured files as per [installation guide](https://tailwindcss.com/docs/installation/using-postcss). However, the tailwind CSS shows no effect and does not apply any style to the page. Codes of config files can be found below:

**postcss.config.js**

```
   module.exports = {
      plugins: {
        tailwindcss: {},
        autoprefixer: {},
      },
    }
```
**tailwind.config.js**

```
  module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
**globals.css**

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

_Note: I am running tailwind in my nodejs project and it has been installed by npm and it is in node modules folder, but the tailwind config and postcss config files are not in any sub folders._

**Comment by ballerolu** [comment]
Hey! I had this same issue for a couple days. I just tried adding `<script src="https://cdn.tailwindcss.com"></script>` to the `head` tag in `index.html` file and the styles started to apply.

**Reply by adcrown1503** [reply]
Perfect Answer
